superbench/benchmarks/micro_benchmarks/ib_validation_performance.py

```diff
@@ -28,67 +27,79 @@ def __init__(self, name, parameters=''):
             'ib_write_bw', 'ib_read_bw', 'ib_send_bw', 'ib_write_lat', 'ib_read_lat', 'ib_send_lat'
         ]
         self.__patterns = ['one-to-one', 'one-to-many', 'many-to-one']
-        self.__config_path = os.getcwd() + '/config.txt'
+        self.__config_path = os.path.join(os.getcwd(), 'config.txt')
         self.__config = []
 
     def add_parser_arguments(self):
         """Add the specified arguments."""
         super().add_parser_arguments()
 
         self._parser.add_argument(
-            '--ib_index',
-            type=int,
-            default=0,
+            '--ib_dev',
+            type=str,
+            default='mlx5_0',
+            required=False,
+            help='The IB device, could be mlx5_0 or mlx5_$LOCAL_RANK.',
+        )
+        self._parser.add_argument(
+            '--gpu_dev',
+            type=str,
+            default=None,
+            required=False,
+            help='The GPU device, could be 0 or $LOCAL_RANK',
+        )
+        self._parser.add_argument(
+            '--numa_dev',
+            type=str,
+            default=None,
             required=False,
-            help='The index of ib device.',
+            help='The NUMA node to bind, could be 0 or $((LOCAL_RANK/2))',
```

Comment:
say we have a node with 8 NICs, 8 GPUs, and 4 NUMA nodes, to get the full bandwidth of parallel ib_write_bw test, we need to assign the corresponding GPU to each NIC, and bind the process to correct NUMA (if all 8 processes are bound to numa0 or no bind, there may exist regression in bw when they run simultaneously).

here's an example how we set the config for above case: https://github.com/microsoft/superbenchmark/blob/458a632db91eae2b879a8dc12d0d8156f6ad3eb6/superbench/config/azure_ndv4.yaml#L170-L177

Reply:
Thanks for your explanation. 
It's true for ndv4, where each node has 8 NICs, 8 GPUs and 4 NUMA nodes. My little concern is if it's generic to say LOCAL_RANK/2 in the help message.

Reply:
the help message just aims to give a hint that the device id could be set in this way, it can be `$LOCAL_RANK` or some calculations for `$LOCAL_RANK`.
do you have any suggestions on this help message?

Reply:
updated to `e.g., 0, $LOCAL_RANK, $((LOCAL_RANK/2)), etc.`